Makefile.am

```diff
@@ -188,7 +188,13 @@ EXTRA_DIST+= \
 	scripts/maint/practracker/testdata/header.h 	\
 	scripts/maint/practracker/testdata/not_c_file	\
 	scripts/maint/practracker/test_practracker.sh   \
-	scripts/maint/practracker/util.py
+	scripts/maint/practracker/util.py               \
+	scripts/coccinelle/apply.sh                     \
+    scripts/coccinelle/check_cocci_parse.sh         \
```

Comment:
nit: This list now has mixed tab and space indentation.

Reply:
nit: also, the test_practracker.sh lines and below use spaces before the backslashes instead of tabs like the rest of the lines there do